main.go

```diff
@@ -118,6 +119,9 @@ func main() {
 	pflag.StringVar(&cfg.Controller.ImagePullSecrets, "image-pull-secrets", "", "Secrets used for pulling the Docker image from a private registry")
 	handleFatalError(viper.BindPFlag("controller.imagePullSecrets", pflag.Lookup("image-pull-secrets")))
 
+	pflag.IntVar(&cfg.Controller.ExpiredDisruptionGCDelay, "expired-disruption-gc-delay", 300, "Seconds after a disruption expires before being automatically deleted")
```

Comment:
Can you also add this configuration to the [configmap](https://github.com/DataDog/chaos-controller/blob/main/chart/templates/configmap.yaml) and [values.yaml](https://github.com/DataDog/chaos-controller/blob/main/chart/values.yaml) files of the chart so it can be configured through it please?